File: legacy/Romaji.java
package me.kotlia;

import com.ibm.icu.text.Transliterator;

public class Romaji {
    public static String hiraganize(String text) {
        return Transliterator.getInstance("Latin-Hiragana").transliterate(text);
    }
    public static String romanize(String text) {
        return Transliterator.getInstance("Hiragana-Latin").transliterate(text);
    }
}

File: src/main/kotlin/me/kotlia/Romaji.kt
package me.kotlia

import com.ibm.icu.text.Transliterator

object Romaji {
    fun hiraganize(text: String): String {
        return Transliterator.getInstance("Latin-Hiragana").transliterate(text)
    }
    fun romanize(text: String): String {
        return Transliterator.getInstance("Hiragana-Latin").transliterate(text)
    }
}